app/src/main/res/drawable/fog.xml

```diff
@@ -0,0 +1,12 @@
+<vector android:height="32dp" android:viewportHeight="96.0"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.fog` appears to be unused
 Unused resources make applications larger and slow down builds.